tests/e2e/vcp_to_csi_syncer.go

```diff
@@ -1429,6 +1430,228 @@ var _ = ginkgo.Describe("[csi-vcp-mig] VCP to CSI migration syncer tests", func(
+		readDataFromRawBlockVolume(namespace, pod.Name, pvc1_devicePath, "This is write to first VCP volume")
+
+		ginkgo.By("Creating VCP PVC pvc2 with raw block volumemode post migration")
+		pvc2, err := fpv.CreatePVC(client, namespace, vcpPvcSpec)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred(), fmt.Sprintf("Failed to create pvc with err: %v", err))
+		vcpPvcsPostMig = append(vcpPvcsPostMig, pvc2)
+
+		ginkgo.By("Waiting for all claims created post migration to be in bound state")
+		vcpPvsPostMig, err = fpv.WaitForPVClaimBoundPhase(client, vcpPvcsPostMig, framework.ClaimProvisionTimeout)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		ginkgo.By("Verify annotations on PV/PVCs created post migration")
+		waitForMigAnnotationsPvcPvLists(ctx, client, vcpPvcsPostMig, vcpPvsPostMig, false, migrationEnabledByDefault)
+
+		ginkgo.By("Wait and verify CNS entries for all CNS volumes created post migration " +
+			"along with their respective CnsVSphereVolumeMigration CRDs")
+		verifyCnsVolumeMetadataAndCnsVSphereVolumeMigrationCrdForPvcs(ctx, client, vcpPvcsPostMig)
+
+		dep1, err = client.AppsV1().Deployments(namespace).Get(ctx, dep1.Name, metav1.GetOptions{})
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		pods, err = fdep.GetPodsForDeployment(client, dep1)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		pod = pods.Items[0]
+		rep := dep1.Spec.Replicas
+		*rep = 0
+		dep1.Spec.Replicas = rep
+		ginkgo.By("Scale down deployment to 0 replica")
+		dep1, err = client.AppsV1().Deployments(namespace).Update(ctx, dep1, metav1.UpdateOptions{})
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		err = fpod.WaitForPodNotFoundInNamespace(client, pod.Name, namespace, pollTimeout)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		pvclaims := []*v1.PersistentVolumeClaim{pvc1, pvc2}
+		var volumeDevices = make([]v1.VolumeDevice, len(pvclaims))
+		var volumes = make([]v1.Volume, len(pvclaims))
+		for index, pvclaim := range pvclaims {
+			volumename := fmt.Sprintf("volume%v", index+1)
+			volumeDevices[index] = v1.VolumeDevice{Name: volumename, DevicePath: "/mnt/" + volumename}
+			volumes[index] = v1.Volume{Name: volumename, VolumeSource: v1.VolumeSource{
+				PersistentVolumeClaim: &v1.PersistentVolumeClaimVolumeSource{ClaimName: pvclaim.Name, ReadOnly: false}}}
+		}
+		dep1, err = client.AppsV1().Deployments(namespace).Get(ctx, dep1.Name, metav1.GetOptions{})
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		dep1.Spec.Template.Spec.Containers[0].VolumeDevices = volumeDevices
+		dep1.Spec.Template.Spec.Volumes = volumes
+		*rep = 1
+		dep1.Spec.Replicas = rep
+		ginkgo.By("Update deployment to use pvc1 and pvc2")
+		dep1, err = client.AppsV1().Deployments(namespace).Update(ctx, dep1, metav1.UpdateOptions{})
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		err = fdep.WaitForDeploymentComplete(client, dep1)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		framework.Logf("sleep for 1 min...")
```

Comment:
can you add comments here on why we need a sleep call here

Reply:
removed sleep